src/guides/v2.4/graphql/interfaces/configurable-product.md

```diff
@@ -20,17 +20,9 @@ The `ConfigurableProduct` object contains the following attributes:
 Attribute | Type | Description
 --- | --- | ---
 `configurable_options` | [[ConfigurableProductOptions]](#configProdOptions) | An array of linked simple product items
+`configurable_product_options_selection(configurableOptionValueUids: [ID!])` | [ConfigurableProductOptionsSelection](#ConfigurableProductOptionsSelection) | Contains media gallery items and other details about selected configurable product options as well as details about remaining selectable options. We recommend filtering by one or more `uid` values to prevent loading a large amount of media gallery data
```

Comment:
```suggestion
`configurable_product_options_selection(configurableOptionValueUids: [ID!])` | [ConfigurableProductOptionsSelection](#ConfigurableProductOptionsSelection) | Contains media gallery items and other details about selected configurable product options and remaining selectable options. Magento recommends you filter by one or more `uid` values to prevent loading a large amount of media gallery data
```